modules/manage/pages/manage-settings/general-settings.adoc

```diff
@@ -226,7 +226,7 @@ The number can be increased on multi-core machines.
 The default is 0.
 
 * *Max Rollback Points*.
-The maximum number of committed rollback points.
+The maximum number of xref:learn:services-and-indexes/indexes/index-replication.adoc#index-rollback[committed rollback points].
```

Comment:
This setting is no longer on the UI. The default has been changed to 2 as well.

Reply:
Done